Bottom: 15d9afef56f9785983e2b3ae11436b806231657a
Top:    e952549640e99fa10908312efda036a7e057b440
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 19:26:31 +0100

crosshair.c: Don't snap to an element's mark in rat-draw mode.


---

diff --git a/src/crosshair.c b/src/crosshair.c
index b5e35f5..67613b6 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -862,7 +862,7 @@ FitCrosshairIntoGrid (LocationType X, LocationType Y)
   snap_data.y = y;
 
   ans = NO_TYPE;
-  if (PCB->RatDraw || TEST_FLAG (SNAPPINFLAG, PCB))
+  if (!PCB->RatDraw && TEST_FLAG (SNAPPINFLAG, PCB))
     ans = SearchScreenGridSlop (Crosshair.X, Crosshair.Y,
                                 ELEMENT_TYPE, &ptr1, &ptr2, &ptr3);
